src/content/docs/errors-inbox/errors-inbox.mdx

```diff
@@ -165,3 +166,24 @@ To use CodeStream's Open in IDE integration with your APM stack trace errors, us
 Once set up, you can jump from an error group directly to the offending code in your IDE by clicking the **Open in IDE** button.
 
 Learn more [here](docs/codestream/start-here/codestream-new-relic/#apm).
+
+## Connect an inbox to Jira [#jira]
+
+Connect errors inbox to Jira to easily create tickets for your errors, allowing for faster collaboration and resolution. Templates allow you to quickly create a ticket containing error details and links directly to the stack trace and APM for quick access and resolution. 
+
+We store a link to the the ticket alongside the associated error group for a period of time. If the error occurs again within that period, you can easily access closed associated tickets for context.
```

Comment:
Is there a way to characterize how long you mean by "period of time?"

Also, this is just a readability suggestion: "easily access closed associated tickets" is succinct but it's a mouthful and I had to read it a couple of times to understand. Perhaps "easily access associated tickets that are closed" is a little smoother.